mmdeploy/codebase/mmagic/deploy/super_resolution_model.py

```diff
@@ -121,30 +121,33 @@ def forward(self,
             get_root_logger().warning(f'expect input device {self.device}'
                                       f' but get {lq.device}.')
         lq = lq.to(self.device)
+
         batch_outputs = self.wrapper({self.input_name:
-                                      lq})[self.output_names[0]].to('cpu')
+                                      lq})[self.output_names[0]].to(self.device)
```

Comment:
Generally, we do `destruct` on CPU devices. So, there is no need to change `cpu` to `self.device`.

Reply:
> Generally, we do `destruct` on CPU devices. So, there is no need to change `cpu` to `self.device`.

OK, I will change that back.